scalereg/reg6/staff/views.py

```diff
@@ -33,9 +33,9 @@ def CheckIn(request):
       })
 
   attendees = []
-  if request.POST['verisign']:
+  if request.POST.get('verisign'):
```

Comment:
BTW, I think request.POST['key_name'] should still work in 1.11 just like before. If there is a MultiValueDictKeyError, that is likely a separate issue. We can deal with that in a separate PR and add the appropriate 'key_name' in request.POST check.

Reply:
The distinction is just that request.POST[] will return a keyerror or multidict error if the variable in'st present.  request.POST.get returns NONE or a default value if you provide one.   

In one or two places (i can't recall where specifically) we were seeing failures because of headers that were optional.  Can pull this out and make a 2nd PR, but this seems like the behavior we'd want.

Reply:
Let's leave it alone. Exceptions being throwing is a good thing. It means some places are not checking the keys like they should. Converting them to get() is just masking the failure. As an example of a check, look at CheckPaymentAmount() in reg6/views.py. Before accessing request.POST['AMOUNT'], the code first calls CheckVars() to make sure the 'AMOUNT' key exists. There is no need to convert that to get().

Also, note that the staff pages may be less robust since they are not publicly accessible.